src/LanguageServer/Protocol/Handler/Diagnostics/DiagnosticSources/NonLocalDocumentDiagnosticSource.cs

```diff
@@ -26,7 +26,7 @@ public override async Task<ImmutableArray<DiagnosticData>> GetDiagnosticsAsync(
         var service = this.Solution.Services.GetRequiredService<IDiagnosticAnalyzerService>();
         var diagnostics = await service.GetDiagnosticsForIdsAsync(
             Document.Project, Document.Id, diagnosticIds: null, _shouldIncludeAnalyzer,
-            includeLocalDocumentDiagnostics: false, includeNonLocalDocumentDiagnostics: true, cancellationToken).ConfigureAwait(false);
+            includeLocalDocumentDiagnostics: false, cancellationToken).ConfigureAwait(false);
```

Comment:
no change here.

Reply:
If non local diagnostics are always being returned, it seems like we need to delete this source?  This was a separate source because the normal diagnostic sources called `GetDiagnosticsForSpan` which did not report these.  But if it is now reporting them, we'll be reporting them both from the other document sources and this one?